xarray/core/variable.py

```diff
@@ -29,6 +29,9 @@
     pass
 
 
+basic_indexing_types = integer_types + (slice,)
```

Comment:
Per PEP8, module level constants should be spelled in all capital letters, i.e., `BASIC_INDEXING_TYPES`